source/renderer/app/stores/StakingStore.js

```diff
@@ -239,6 +241,8 @@ export default class StakingStore extends Store {
       this.numberOfStakePoolsFetched > 0
     ) {
       this.cyclesWithoutIncreasingStakePools++;
+    } else {
```

Comment:
@daniloprates this is just FYI - no response required.
I believe this was a flaw in your original implementation.
We need to make sure the tracker is switched off only in case we don't see an increase in stake pools count for a six **consecutive** checks. Without this change we don't count consecutive ones at all - we count all of them.